.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,18 @@
+# For more information see: https://github.com/marketplace/actions/commit-linter
+
+name: 'Lint the Commit Messages'
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  commitlint:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: 'actions/checkout@v2'
+        with:
+          fetch-depth: 0
+      - uses: 'wagoid/commitlint-github-action@v3'
```

Comment:
Could the `configFile` be specified, please? I know the default is correct. I prefer the clarity.

Reply:
Oh right, I also prefer clarity, actually I didn't know, we could provide `configFile`, I just saw on the doc : https://github.com/marketplace/actions/commit-linter

Thanks, I'll also update my repos with that.